uninstall.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * Distributor uninstall script.
+ *
+ * @package distributor
+ * @since   x.x.x
+ */
+
+// phpcs:disable
+
+if ( ! defined( 'WP_UNINSTALL_PLUGIN' ) ) {
+	die;
+}
+
+/*
+ * Only remove ALL product and page data if DT_REMOVE_ALL_DATA constant is set to true in user's
+ * wp-config.php. This is to prevent data loss when deleting the plugin from the backend
+ * and to ensure only the site owner can perform this action.
+ */
+if ( defined( 'DT_REMOVE_ALL_DATA' ) && true === DT_REMOVE_ALL_DATA ) {
```

Comment:
This will only remove the data from the current site (the network base site on the network tab) but the data will remain on sub-sites.

It might be possible to loop through the sites on small networks.

Reply:
@peterwilsoncc, after discussing with @jeffpaul, we decided to take care of the main website. It will be helpful for users to be aware of this. 

@jeffpaul Do you have thoughts?

Reply:
I think that so long as we've presented the admin uninstalling Distributor with an "are you sure" sort of message on the impacts of fully removing Distributor data, then we should look for a reliable and performant, if slow and long-running, approach to clearing that data across all sites on a network.